pkgs/development/ocaml-modules/luv/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildDunePackage, fetchurl
+, ctypes, result
+, alcotest
+, file
+}:
+
+buildDunePackage rec {
+  pname = "luv";
+  version = "0.5.7";
+  useDune2 = true;
+
+  src = fetchurl {
```

Comment:
Can we use fetchFromGitHub?

Reply:
No, you can’t use `fetchFromGitHub` to download that file.